app/local.env

```diff
@@ -63,8 +63,12 @@ HIDE_SQL_PARAMETER_LOGS=TRUE
 # add them to this file to avoid mistakenly
 # committing them. Set these in your shell
 # by doing `export AWS_ACCESS_KEY_ID=whatever`
+# if you are running the app directly, or
+# in your `app/.env` if you are running the
+# app in a Docker container
```

Comment:
Threading arbitrarily, - we might also need a change to the `load_local_env_vars` method which handles getting the `local.env` file into your env vars when you run outside of docker. Right now that automatically gets called for unit tests in the conftest file:

https://github.com/navapbc/template-application-flask/blob/main/app/tests/conftest.py#L21

I think if we just adjusted `load_local_env_vars` to do "if .env exists, load it before the local.env file" bit - then it would work.

Reply:
Alternatively - we _could_ get rid of load_local_env_vars entirely and more accurately document how to do that properly when running outside of docker. Maybe I'll take that on next week.